Resources/Prototypes/Entities/Objects/Consumable/Food/meals.yml

```diff
@@ -98,11 +98,25 @@
   id: FoodMealNachos
   description: Chips from Space Mexico.
   components:
+  - type: Food
+    trash:
+    - FoodPlateSmall
   - type: FlavorProfile
     flavors:
       - nachos
   - type: Sprite
     state: nachos
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 15
+        reagents:
+        - ReagentId: Nutriment
```

Comment:
Shouldn't they have a little less nutriment/vitamin than the cheese nachos since they have less ingredients?

Reply:
You're right. I pulled the values from the /tg/station wiki for consistency without doing any numbers. Nachos are now 4/1, cheesy 6/3, and cuban is 8/5 for having produce in it. Also fixed Cuban not having enough volume for its reagents, changed flavor from cheesy to tomato, and made it use one less pepper to fit better nutritionally there too.